meta-lmp-base/recipes-devtools/mcumgr/mcumgr_git.bb

```diff
@@ -5,7 +5,7 @@ LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
 
 GO_IMPORT = "github.com/apache/mynewt-mcumgr-cli/mcumgr"
-SRC_URI = "git://github.com/apache/mynewt-mcumgr-cli;protocol=https"
+SRC_URI = "git://github.com/apache/mynewt-mcumgr-cli;protocol=https;branch=master"
```

Comment:
is it worth creating a variable? Something such as: `branch=${BRANCH}`

There is a empty line between the 2 signed-off-by lines in this commit log

Reply:
This cherry-picked patch, I don't want to change it in that manner.

Thanks, the empty line was removed.